tasks/strings/strings.js

```diff
@@ -4,23 +4,38 @@
  * @param {string} str
  */
 function reverseString(str) {
+  if (typeof str != 'string') {
+    return 'This is not a string!';
+  }
+  return (str === '') ? '' : reverseString(str.substr(1)) + str.charAt(0);
 }
 
+
 /**
  * 1. Calculate a century by given year
  * if a number is passed
 * @param {number} year
 */
 function centuryFromYear(year) {
+  var a = Math.ceil(year/100);
```

Comment:
rewrite in one-line:
return Math.ceil(...)

Reply:
`function centuryFromYear(year) {
  return Math.ceil(year / 100);
} `